src/games/Lexible/models/ClientModel.ts

```diff
@@ -140,22 +153,33 @@ export class LexibleClientModel extends ClusterfunClientModel  {
     // -------------------------------------------------------------------
     // submitWord 
     // -------------------------------------------------------------------
-    submitWord() {
+    async submitWord() {
         if(this.letterChain.length === 0) {
             Logger.warn("WEIRD:  should have been letters in the letter chain")
             return;
         }
 
-        const submissionData: WordSubmissionData = {
+        const submissionData: LexibleWordSubmissionRequest = {
             letters: this.letterChain.map(l => ({letter: l.letter, coordinates: l.coordinates}))
         }
 
-        this.sendAction(LexiblePlayerAction.WordSubmit, submissionData)
+        const response = await this.session.request(LexibleSubmitWordEndpoint, this.session.presenterId, submissionData)
```

Comment:
That's mostly correct, yes. I can see some games where clients talk to each other directly, but since most messages are for the presenter, I should add a requestPresenter API. I'll do that in a separate branch.